### ISSUE TITLE
[Designer] Bug 33489500: [Activity Update with Label]: Screen reader is not announcing the 'Expand/Collapse' state of "Set Due Date" and "Comments" buttons under 'Activity Update with Label' Adaptive Card.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7625]
### Problem Description

[Bug 33489500](https://microsoft.visualstudio.com/OS/_workitems/edit/33489500): [Activity Update with Label]: Screen reader is not announcing the 'Expand/Collapse' state of "Set Due Date" and "Comments" buttons under 'Activity Update with Label' Adaptive Card.

Test Environment:

Application: Adaptive Cards
OS version: Dev (OS Build 21387.1)

Application Version: 2.7.2105.11001

Screen Reader: Narrator

Repro steps:

1. Launch the Adaptive Card application and narrator.

2. Adaptive Card UWP Visualizer screen starts appearing. 

3. Navigate to Activity Update with Label button by using Tab key and activate it.

4. Activity Update with Label adaptive card will be appeared on right sight of the pane.

5. Navigate to Activity Update with Label card by using Tab key.

6. Navigate to "Set Due Date" and "Comments" buttons with Tab key and observe the issue.

Actual Result:
Screen reader is not announcing the 'Expand/Collapse' state of "Set Due Date" and "Comments" buttons under 'Activity Update with Label' Adaptive Card. Screen Reader is only announcing 'Set Due date button'.

Expected Result:
Screen reader should announce the 'Expand/Collapse' state of "Set due date" and "Comments" buttons under 'Activity Update with Label'. Adaptive Card. Screen reader should announce as, "Set due date button Expanded/Collapsed".

User Experience:
User depending on screen reader might get affected if screen reader does not pass State information about the control. Screen reader user will not able to know the property of that control and will face problem while interacting with it.

Observation: Issue also repro with NVDA(2020.4).

Note: Same issue is also repro with "Set Due Date" button under ActivityUpdate.json card,

### Card JSON

```JSON
Designer Bug. Repro on UWP Adaptive Card visualizer
```

**Comment by vagpt** [comment]
There is no user impact on the screen reader user, hence closing this issue.
